configuration/configure-automatic-upgrades.adoc

```diff
@@ -1,18 +1,21 @@
 [id="configure-automatic-upgrades"]
-= Configuring automatic upgrades for secured clusters
+= Configuring automatic upgrades for manifest-installed secured clusters
 include::modules/common-attributes.adoc[]
 :context: configure-automatic-upgrades
 
 toc::[]
 
 [role="_abstract"]
-You can automate the upgrade process for each secured cluster and view the upgrade status from the {product-title-short} portal.
+If you installed {product-title} by using the manifest installation method, also known as the `roxctl` CLI method, you can automate the upgrade process for each secured cluster. This process is also called the _legacy installation method_ in the {product-title-short} portal. You can also view the upgrade status from the {product-title-short} portal.
```

Comment:
> you can automate the upgrade process for each secured cluster. This process is also called the _legacy installation method_

This is confusing a bit. "This process" does not refer to the upgrade process, but to the installation method. Here is my suggestion:

```suggestion
If you installed {product-title} by using the manifest installation method, also known as the `roxctl` CLI method or the _legacy installation method_, you can automate the upgrade process for each secured cluster. You can also view the upgrade status from the {product-title-short} portal.
```